wordpress-mobile/WordPress-iOS
# [Site creation] Error when submitting a request to create a new site

Fixes #10908 or at least provides a partial workaround. As discussed in issue #10908  this might not be the actual solution

To test:
- Checkout the branch. Attempt to create a new wordpress.com site. The site should be created

![simulator screen shot - iphone xr - 2019-01-29 at 12 12 23](https://user-images.githubusercontent.com/2722505/51886977-ebed2a80-23cc-11e9-97de-a282ec1d98e4.png)
